expenses-manager-frontend/src/main/frontend/src/app/sidebar/sidebar.component.scss

```diff
@@ -0,0 +1,43 @@
+.sidebar{
```

Comment:
as sugera sa redenumim .sidebar in b-sidebar.
Poti arunca o privire aici: https://github.com/nikita-kit/nikita-css